kubeflow/trainer/backends/container/utils_test.py

```
@@ -0,0 +1,109 @@
+    try:
+        trainer = types.CustomTrainer(
+            func=simple_train_func,
+            packages_to_install=test_case.config.get("packages"),
+            pip_index_urls=test_case.config.get("pip_index_urls"),
+        )
+        result = container_utils.build_pip_install_cmd(trainer)
+
+        assert test_case.expected_status == SUCCESS
+        assert result == test_case.expected_output
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)